server/model/monitor.js

```diff
@@ -463,6 +481,12 @@ class Monitor extends BeanModel {
                     retries++;
                     bean.status = PENDING;
                 }
+
+                // Do not change this text!
+                // Condition below and in api-router depends on it
```

Comment:
Can you use some global constant for this text in `monitor.js` and `api-router.js`

Reply:
Added `MONITOR_DOWN_DEGRADED` variable to utils.ts/.js